Digizuite/Extensions/MetadataValueServiceExtensions.cs

```diff
@@ -417,30 +421,31 @@ private static ComboValue ParseEditComboValueValue(ComboValueResponse value)
         }
 
         private static async Task<(TField, TData)> GetSingleField<TField, TData>(this IMetadataValueService service,
-            int assetItemId, int metafieldLabelId, 
+            int assetItemId, int metafieldLabelId,
             CancellationToken cancellationToken, Func<TData> getDefaultValue)
             where TField : MetaFieldResponse
             where TData : MetadataResponse
         {
             var requestBody = new GetMetadataRequest
             {
-                ItemIds = new List<int>{assetItemId},
-                LabelIds = new HashSet<int>{metafieldLabelId}
+                ItemIds = new List<int> { assetItemId },
+                LabelIds = new HashSet<int> { metafieldLabelId }
             };
 
             var response = await service.GetRawMetadata(requestBody, cancellationToken).ConfigureAwait(false);
 
-            var field = (TField) response.Fields.Single();
+            var field = (TField)response.Fields.Single();
 
             TData value;
             if (response.Values.Count == 0)
             {
                 value = getDefaultValue();
+                value.MetafieldId = field.MetafieldId;
```

Comment:
I don't get why we suddenly set `MetafieldId` here? The values are label specific?

Reply:
So we don't need it?
I though we want to get rid of labelid in values, to make value language agnostic? so adding MetafieldId is a first step. 
Or maybe I'm wrong here

Reply:
@zlepper talked to Magnus, yeah , we don't need that MetafieldId for values. at least in a foreseeable future :)